core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigLoader.java

```diff
@@ -265,6 +266,28 @@ static DriverConfigLoader fromMap(@NonNull OptionsMap source) {
     return new MapBasedDriverConfigLoader(source, source.asRawMap());
   }
 
+  /**
+   * Composes two existing config loaders to form a new one.
+   *
+   * <p>When the driver reads an option, the "primary" config will be queried first. If the option
+   * is missing, then it will be looked up in the "fallback" config.
+   *
+   * <p>All execution profiles will be surfaced in the new config. If a profile is defined both in
+   * the primary and the fallback config, its options will be merged using the same precedence rules
+   * as described above.
+   *
+   * <p>The new config is reloadable if at least one of the input configs is. If you invoke {@link
+   * DriverConfigLoader#reload()} on the new loader, it will reload whatever is reloadable, or fail
+   * if nothing is. If the input loaders have periodic reloading built-in, each one will reload at
+   * its own pace, and the changes will be reflected in the new config.
+   */
+  @NonNull
+  static DriverConfigLoader compose(
+      @NonNull DriverConfigLoader primaryConfigLoader,
+      @NonNull DriverConfigLoader fallbackConfigLoader) {
```

Comment:
Because it's overkill IMHO.

Frankly I already have doubts about composing two loaders (per my previous comment). Composing more than two sounds even more wrong.

You can always chain `compose()` calls.